docs/conf.py

```diff
@@ -136,7 +137,7 @@ def __getattr__(cls, name):
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = []
+exclude_patterns = ["_build", "**.ipynb_checkpoints"]
```

Comment:
Are these necessary?

Reply:
Hmm interesting! AFAIK we should be starting from clean every time, but @philipc2 might know that better

Reply:
IMHO when you are developing it's a little inconvenient to have to remove `_build` every time, this just alleviates the need for that.